homotopy.tex

```diff
@@ -1003,7 +1003,7 @@ \section{Fiber sequences and the long exact sequence}
   \begin{enumerate}
   \item The fiber of $f^{(1)}\defeq \proj1 : \hfib f {y_0} \to X$ is equivalent to $\Omega Y$.\label{item:fibseq1}
   \item Similarly, the fiber of $f^{(2)} : \Omega Y \to \hfib f {y_0}$ is equivalent to $\Omega X$.\label{item:fibseq2}
-  \item Under these equivalences, the map $f^{(3)} : \Omega X\to \Omega Y$ is identified with $\Omega f\circ\rev{(\blank)}$.\label{item:fibseq3}
+  \item Under these equivalences, the pointed map $f^{(3)} : \Omega X\to \Omega Y$ is identified with $\Omega f\circ\rev{(\blank)}$ (here we treat path inversion as pointed map of type $\Omega X\to \Omega X$).\label{item:fibseq3}
```

Comment:
I don't know. I thought it might be useful, because we haven't introduced path inversion as a pointed map yet. But I'm okay with removing it.